packages/opentelemetry-resources/src/platform/node/detectors/AwsEc2Detector.ts

```diff
@@ -28,10 +28,17 @@ import { ResourceDetectionConfigWithLogger } from '../../../config';
 class AwsEc2Detector implements Detector {
   /**
    * See https://docs.aws.amazon.com/AWSEC2/latest/UserGuide/instance-identity-documents.html
-   * for documentation about the AWS instance identity document endpoint.
+   * for documentation about the AWS instance identity document
+   * and standard of IDMSv2.
    */
-  readonly AWS_INSTANCE_IDENTITY_DOCUMENT_URI =
-    'http://169.254.169.254/latest/dynamic/instance-identity/document';
+  readonly HTTP_HEADER = 'http://';
```

Comment:
This is probably not a great name since HTTP Header already has its own meaning. It looks like you're looking for the [URL Scheme](https://url.spec.whatwg.org/#url-representation).

It looks like this is also only used in tests, so why is it defined here?

Reply:
Oh here is a historical problem. Initially we use Url in order to connect with AWS identity document. And current version we use host and path separately.
I will move this constant to test code, thank you for pointing out!